pkg/gameplay/game_test.go

```diff
@@ -118,11 +130,13 @@ func TestComputeGameStats(t *testing.T) {
 	is.NoErr(err)
 
 	ctx := context.WithValue(context.Background(), ConfigCtxKey("config"), &DefaultConfig)
-	stats, err := computeGameStats(ctx, hist, gameReq, variantKey(req), ustore)
+	// Fix this once list separation is complete
```

Comment:
does this comment still apply? what needs to be fixed?

Reply:
We need to implement AddListItem and GetListItems